lang/python/python-markdown/Makefile

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (C) 2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-markdown
+PKG_VERSION:=3.1.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=Markdown-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://pypi.python.org/packages/source/m/markdown/
+PKG_HASH:=2e50876bcdd74517e7b71f3e7a76102050edec255b3983403f1a63e7c8a41e7a
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/Markdown-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE.md
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-markdown
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Markdown implementation in Python
+  URL:=https://python-markdown.github.io/
+  DEPENDS:= \
+	+python3-light \
+	+python3-setuptools
```

Comment:
this installs `setuptools` on the target;
i am not sure that is required; it seems required only during build
would removing this also work?

Reply:
It appears to [use pkg_resources find extensions](https://github.com/Python-Markdown/markdown/blob/3.1.1/markdown/core.py#L157), so it does seem to need setuptools installed at run-time. (It also uses it to [normalize it's version number](https://github.com/Python-Markdown/markdown/blob/3.1.1/markdown/__meta__.py#L26).)

Reply:
I would suggest adding `python3-logging` as a dependency, since you're getting an import error and it does appear to [use logging directly](https://github.com/Python-Markdown/markdown/search?q=logging&unscoped_q=logging).

Reply:
If I don't have any python3 package and install python3-setuptools it installs:
python3-base, python3-light, python3-pydoc, python3-email, python3-cgi, python3-cgitb, python3-decimal, python3-xml, python3-unittest, python3-ncurses, python3-ctypes, python3-multiprocessing- python3-codecs, python3-gdbm, python3- sqlite3, python3-logging, python3-distutils, python3-distutils, python3-openssl, python3-dbm, python3-lzma, python3-asyncio.

So yes, python3-logging it gets pulled by python3-setuptools, but I don't mind to specify there python3-logging as well.